chapters.md

```diff
@@ -3,31 +3,82 @@
 
 # Chapters
 
-## Edition and Chapter Flags
+The Matroska Chapters system can consist of`Simple Chapters` where a chapter start time is used as marker in the timeline only. The system can be more complex with `Ordered Chapters` where a chapter end time stamp is additionally used or much more complex with `Linked Chapters`. The Matroska Chapters system can also have a menu structure, borrowed from the DVD menu system, or have it's own Native Matroska menu structure.
```

Comment:
- missing space in "of\`Simple Chapters\`"
- is `Simple Chapters` intended as a formal term? Does it mean non `Ordered Chapters`?
- the qualities of the chapters should be reference within an edition rather than related to the `Matroska Chapters system` since a Matroska may include all of these types of chapters simultaneously.